pkg/web/api/v1/transaction.go

```diff
@@ -69,6 +69,7 @@ type SecureEnvelope struct {
 }
 
 type Envelope struct {
+	EnvelopeID  string                   `json:"envelope_id,omitempty"`
```

Comment:
I added the EnvelopeID to the top level -- instead of calling it TransactionID I called it EnvelopeID to avoid confusion with the Transaction and Pending payloads; but in this context they are synonyms (I know there is a lot of term overload, sorry).